generate_workflow.py

```diff
@@ -105,7 +105,7 @@ def tuxsuite_setups(job_name, tuxsuite_yml, repo, ref):
                 },
                 {
                     "name": "tuxsuite",
-                    "run": "tuxsuite plan --git-repo {} --git-ref {} --job-name {} --json-out builds.json {}{} || true".format(repo, ref, job_name, tuxsuite_yml, patch_series)
+                    "run": "tuxsuite plan --git-repo {} --git-ref {} --job-name {} --json-out builds.json {}{} || true".format(repo, ref, job_name, patch_series, tuxsuite_yml)
```

Comment:
add whitespace between the `{}{}`

Reply:
I did it this way to avoid two spaces when there is no patch series but since it is purely cosmetic, I can just do what you have requested.

Reply:
Done.